time-test-harness/README.md

```diff
@@ -8,7 +8,7 @@
   
 This test harness lets you easily simulate the behavior that a consumer of the `viv.time` library would see. 
 
-(`viv.time` library documentation: https://bixbydevelopers.com/dev/docs/dev-guide/developers/library.datetime)
+(`viv.time` library documentation: https://bixbydevelopers.com/dev/docs/reference/ref-topics/library.datetime#datetime)
```

Comment:
```suggestion
(`viv.time` library documentation: https://bixbydevelopers.com/dev/docs/reference/ref-topics/library.datetime)
```

Could you remove the `#datetime` id tag at the end? This is to keep the same functionality as before with going to the top of the viv.time reference page.

Reply:
I already removed.